_posts/2020-03-12-Steering-Group-executive-summary,-action-items,-and-minutes.md

```diff
@@ -12,22 +12,22 @@ Attended: Franklin Feingold, Melanie Ganz, Guiomar Niso, Robert Oostenveld, Kirs
 ### Executive Summary
 
 We discussed the channels through which to share updates to the BIDS specification.
-An issue was opened to start crowdsourcing and collecting this information.
-We will be using this to share the release of v1.2.2 and to request the final rounds of feedback on the Genetic information extension.
+[An issue](https://github.com/bids-standard/bids-specification/issues/415) was opened to start crowdsourcing and collecting this information.
+We will be using this list of channels to share the release of v1.2.2 and to request the final rounds of feedback on the Genetic information extension.
 
-We discussed generating an example slide deck that  community members can  use (or pick from) when they want to present BIDS.
+We discussed generating an example slide deck that community members can  use (or pick from) when they want to present BIDS.
 
 We discussed considerations for when to open a repository under the BIDS standard organization on GitHub.
-There should be multiple people committed to the repository (otherwise it is better of under the personal GitHub account), the maintainers should be clearly identifiable (e.g. as a CODEOWNERS file at top level), there should be a code of conduct in the repo that is consistent with that of the BIDS specification, and (as a consequence of the code of conduct) the repository should welcome contributions from others.
+Previously, community members have opened repositories under our BIDS standard organization.
+We want to codify guidelines to govern this.
+The plan is that there should be multiple people committed to the repository (otherwise it is better of under the personal GitHub account), the maintainers should be clearly identifiable (e.g. as a CODEOWNERS file at top level), there should be a code of conduct in the repo that is consistent with that of the BIDS specification, and (as a consequence of the code of conduct) the repository should welcome contributions from others.
```

Comment:
```suggestion
The plan is that there should be multiple people committed to the repository (otherwise it is better of under the personal GitHub account), the maintainers should be clearly identifiable (e.g., as a [CODEOWNERS file](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners) at top level), there should be a code of conduct in the repo that is consistent with that of the BIDS specification, and (as a consequence of the code of conduct) the repository should welcome contributions from others.
```